conversation/v1.js

```diff
@@ -140,6 +142,7 @@ ConversationV1.prototype.message = function(params, callback) {
       url: '/v1/workspaces/{workspace_id}/message',
       method: 'POST',
       json: true,
+      headers: pick(params, ['X-Watson-Origin']),
```

Comment:
I think this will override any default headers (opt-out, token), but that's should really be considered a bug in the request wrapper... Just wanted to point it out here.